core/src/site/resources/tags/script-description.html

```diff
@@ -0,0 +1 @@
+Script tag to be used for adding nonce values to scripts
```

Comment:
let's reword this to give more context
```suggestion
Script tag automatically adds nonces to script blocks - should be used in combination with Strut's CSP Interceptor.
```

Reply:
this got marked as resolved but wasn't :)